scripts/DGSPlanner/DGSPlannerGUI.py

```diff
@@ -145,6 +148,36 @@ def updateUB(self, ol):
         self.updatedOL = True
         self.trajfig.clear()
 
+    def eiWavelengthUpdateEvent(self):
+        if self.masterDict['instrument'] == 'WAND\u00B2':
+            ei = UnitConversion.run('Wavelength', 'Energy', self.masterDict['Ei'], 0, 0, 0, Elastic, 0)
+            offset = ei * 0.01
+            lowerBound = - offset
+            upperBound = offset
+            self.dimensionWidget.set_editMin4(str(lowerBound))
+            self.dimensionWidget.set_editMax4(str(upperBound))
+
+    def instrumentUpdateEvent(self):
+        if self.masterDict['instrument'] == 'WAND\u00B2':
+            # change the ui accordingly
+            self.dimensionWidget.toggleDeltaE(False)
+            self.instrumentWidget.setLabelEi('Input Wavelength')
+            self.instrumentWidget.setEiVal(str(1.488))
+
+            self.instrumentWidget.setGoniometerNames(['s1', 'sgl', 'sgu'])
+            self.instrumentWidget.setGoniometerDirections(['0,1,0', '1,0,0', '0,0,1'])
+            self.instrumentWidget.setGoniometerRotationSense([1, -1, -1])
```

Comment:
When you set these for wand^2, they remain set even if you change the instrument to something else.